modules/ossm-add-project-member-roll-resource-cli.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-add-project-member-roll-resource-cli_{context}"]
+= Adding or removing projects from the service mesh using ServiceMeshMemberRoll resource with the CLI
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+* The `ServiceMeshMemberRoll` resource is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
```

Comment:
Consider either removing the bullets from this list, or adding an introductory sentence that indicates to the reader what the list is describing. What ties together the items in this list? Are these limitations? Features? Additional, but not essential info? I guess as a reader, I'm not really understanding why this info should be a list rather than just sentences.